logs/build.go

```diff
@@ -18,7 +18,7 @@ type buildCmd struct {
 
 func (cmd *buildCmd) Run(ctx context.Context, client *api.Client) error {
 	if cmd.Name == "" && cmd.ApplicationName == "" {
```

Comment:
The check is futile as well, right?

```suggestion
	if cmd.Name == "" {
```